tools/slang-test/main.cpp

```diff
@@ -563,21 +212,21 @@ Result parseOptions(int* argc, char** argv)
         }
         else if( strcmp(arg, "-appveyor") == 0 )
         {
-            g_options.outputMode = kOutputMode_AppVeyor;
+            g_options.outputMode = TestOutputMode::eAppVeyor;
```

Comment:
I sort of agree - that enum class is a bit of a weird combination of behaviors. 
Enum class is used throught the code base in different ways (sometime's it's lower case(!)). I've picked up the e junk from another project, I'll remove that prefix.

Reply:
Yeah, the mix of styles right now is a mess. At one point I tried going with lowerCamelCase because it folllows the big-picture idea that types are UpperCase and values are lowerCase, but it ends up being bad in practice because:

- There is so much precedent that lower-case enum values always require you to do a double-take when reading the code

- The fact that lowerCase enum naming means you can't use a keyword to name the cases turns out to be a bigger annoyance than one might expect (e.g., `Default` is a really common case name to want in an enum).

For now I'd lean toward UpperCamelCase, but as you know I'm also not pushing to do sweeping renaming changes unless they arise naturally as part of edits we have to do anyway.